Description: avoid a couple warnings, fix dependencies
Author: Samuel MARTIN MORO <samuel@peerio.com>
Origin: vendor
Forwarded: not-needed
Last-Update: 2015-11-18
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/package.json	2015-11-16 21:43:28.000000000 -0500
+++ b/package.json	2015-11-18 14:22:44.351122279 -0500
@@ -2,6 +2,11 @@
   "name": "peerio-client",
   "author": "Technologies Peerio Inc.",
   "description": "Secure message and file sharing for everyone. Our worldwide cloud storage lets you work anywhere, while end-to-end encryption keeps your data safe everywhere.",
+  "license": "GPLv3",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/PeerioTechnologies/peerio-client.git"
+  },
   "version": "1.2.0",
   "keywords": [
     "peerio",
@@ -9,6 +14,11 @@
     "encrypted email",
     "file sharing"
   ],
+  "dependencies": {
+    "gulp": "latest",
+    "gulp-shell": "latest",
+    "nw": "latest"
+  },
   "devDependencies": {
     "del": "^1.2.0",
     "gulp": "latest",
